File: client/src/entities/PostCategory.ts
/**
 * 文章分类，每隔文章只会属于一个分类
 */
export default interface PostCategory {

    /**
     * ID
     */
    id?: number;

    /**
     * 文章ID，唯一
     */
    postId: number;

    /**
     * 分类ID，唯一
     */
    categoryId: number;

    /**
     * 创建时间
     */
    createTime: Date;

}

File: server/src/main/java/xyz/esion/yunluoblog/entity/PostCategory.java
package xyz.esion.yunluoblog.entity;

import com.baomidou.mybatisplus.annotation.*;

import java.io.Serializable;
import java.util.Date;
import lombok.Data;

/**
 * @TableName post_category
 */
@TableName(value ="post_category")
@Data
public class PostCategory implements Serializable {

    @TableId(type = IdType.INPUT)
    private Integer id;

    private Integer isDelete;

    @TableField(fill = FieldFill.INSERT)
    private Date createTime;

    @TableField(fill = FieldFill.INSERT_UPDATE)
    private Date updateTime;

    private Integer postId;

    private Integer categoryId;

    @TableField(exist = false)
    private static final long serialVersionUID = 1L;
}